py3status/modules/check_tcp.py

```diff
@@ -3,13 +3,15 @@
 Display status of a TCP port on a given host.
 
 Configuration parameters:
-    cache_timeout: how often to run the check (default 10)
-    format: what to display on the bar (default '{host}:{port} {state}')
-    host: check if tcp port on host is up (default 'localhost')
-    port: the tcp port (default 22)
+    cache_timeout: refresh interval for this module (default 10)
+    format: display format for this module (default '{host}:{port} {state}')
+    host: check host (default 'localhost')
+    port: check port number (default 22)
+    string_down: show string when available (default 'UP')
+    string_up: show string when unavailable (default 'DOWN')
```

Comment:
>it is used as a placeholder replacement
:(

I don't want people having to remember which to use... `icon_` or `state_` or... etc. I was hoping that they can just try `somethinggeneric_on` and it'll work.

Here, I would like if we add couple of lines so if somebody tried `icon_on` (oops, a mistake here), it would be interpreted as `icon_up`. Maybe in post_config_hook?

>We do need to find a solution to these issues at some point

Ya. :-)